static_sources/eateryAttributes.json

```diff
@@ -0,0 +1,67 @@
+{
+  "104-West": {
```

Comment:
is there a reason why we can't do `"104 West"` and `reserve_link`? like is iOS expecting this kind of text format cuz it's kinda weird

Reply:
the indices in the attributes JSON are from the eateries slugs, which we already have stored as a convenient unique identifier - I didn't want to use the name of the eatery because I didn't really want to deal with accent marks. I should probably add a file somewhere that has a list of slugs for each eatery at some point, but for now it's easy enough to find by querying for the slug field in GraphQL (that's how I generally found them)

Reply:
as for the second, I was looking up JSON code practices and found somewhere that camelCase was kind of preferred? It was a random stackoverflow post though so I can just change it to snake_case